frontend/src/core/comments/components/Comment.js

```diff
@@ -7,30 +7,39 @@ import { Localized } from '@fluent/react';
 import './Comment.css';
 
 import type { TranslationComment } from 'core/api';
+import type { UserState } from 'core/user';
 
 import { UserAvatar } from 'core/user'
 
 
 type Props = {|
     comment: TranslationComment,
+    user: UserState,
+    canReview: boolean,
+    deleteComment: (number) => void,
 |};
 
 
-const deleteComment = () => {
-    console.log('Delete')
-}
-
-const shareComment = () => {
-    console.log('Share')
-}
-
 export default function Comment(props: Props) {
-    const { comment } = props;
+    const { comment, canReview, user } = props;
 
     if (!comment) {
         return null;
     }
 
+    // Does the currently logged in user own this Comment?
+    const ownComment = (
+        user && user.username &&
+        user.username === comment.username
+    );
+
+    let canDelete = (canReview || ownComment);
```

Comment:
Eeeeww, `canReview` is a bad name, sorry :/ (it was my doing ;) ). I think what it means is `user can manage this project`, but I don't remember precisely. In the case of history translation, it kinda made sense because it was only used to tell if the user could review or not, but in this case it doesn't really work. Can you please call this something different? Not sure exactly what it should be. The goal is just to make reading this piece of code easier, but having clearer variable names.

Reply:
In order to avoid repeating code, I passed this in as a prop from 'Translation`. Would you prefer I re-write it here with a new name?